pkg/ingester/user_state.go

```diff
@@ -43,13 +43,13 @@ var (
 )
 
 type userStates struct {
```

Comment:
It would be nice to add a comment to this struct, to explain what is it used for. (I know that you haven't introduced it, so feel free to ignore)